plugins/jenkins-backend/src/plugin.test.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { jenkinsPlugin } from './plugin';
+
+describe('jenkins', () => {
+  it('should export the jenkins plugin', () => {
+    expect(jenkinsPlugin).toBeDefined();
+  });
+});
```

Comment:
not sure this test adds value - maybe the others have an opinon here.

Reply:
Thank you for your feedback. 
will be happy to improve as per suggestions.